README.rst

```diff
@@ -37,6 +37,32 @@ Create an Enum-class and pass it as first argument to the Django model EnumField
     class Beer(models.Model):
         style = enum.EnumField(BeerStyle, default=BeerStyle.LAGER)
 
+You can also set default value on your enum class using ``__default__``
+attribute
```

Comment:
I see the convenience of declaring `__default__` just once, but I'm not sure that's an advantage in the long run. I'd rather be explicit than save some keystrokes.

Plus, I would make the argument that the `default` is a property of the field, and therefore belongs to the field declaration. Just in the same spirit as `Field.choices` works. The `Enum` is data, while `default=` is behaviour.